cypress/e2e/default/learn/common-components/search-bar.ts

```diff
@@ -106,6 +106,8 @@ describe('Search bar', () => {
 
     cy.get('.ais-SearchBox-form').submit();
 
-    cy.url('/');
+    cy.location().should(loc => {
+      expect(loc.pathname).to.eq('/learn');
```

Comment:
Sorry about that. This is finicky.

```suggestion
      expect(loc.pathname).to.eq('/learn/');
```